langgraph_bot/insert_bot_data.py

```
@@ -1,79 +1,93 @@
-            print(f"Error inserting '{post.get('title', '<unknown>')}': {e}")
-
-    status = "completed_with_errors" if failed else "completed"
-    return {
-        "status": status,
-        "total": total,
-        "inserted": inserted,
-        "failed": failed,
-    }
\ No newline at end of file
+                supabase.storage.from_(BUCKET).remove([uploaded_path])
+            except Exception as cleanup_error:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)